src/niscope/system_tests/test_system_niscope.py

```diff
@@ -248,17 +248,27 @@ def test_configure_chan_characteristics(session):
     assert 50.0 == session.input_impedance
 
 
-'''
-# TODO(frank): re-add after issue #650 is fixed.
 def test_filter_coefficients():
     with niscope.Session('FakeDevice', False, True, 'Simulate=1, DriverSetup=Model:5142; BoardType:PXI') as session:  # filter coefficients methods are available on devices with OSP
-        assert [1.0, 0.0, 0.0] == session.get_equalization_filter_coefficients(3)
+        assert [1.0] + [0.0] * 34 == session.get_equalization_filter_coefficients() # coefficients list should have 35 items
         try:
             filter_coefficients = [1.0, 0.0, 0.0]
             session.configure_equalization_filter_coefficients(filter_coefficients)
         except niscope.Error as e:
-            assert e.code == -1074135024  # coefficients list should have 35 items
-'''
+            assert "Incorrect number of filter coefficients." in e.description
+            assert e.code == -1074135024
+        filter_coefficients = [0.01] * 35
+        try:
+            # TODO(marcoskirsch): The following should work. It doesn't due to internal NI-SCOPE driver bug 959625.
+            #                     The workaround is to explicitly pass channel "0" rather than the default "" to the driver runtime
+            #                     even though the two should be equivalent on a PXI-5142 (a 1 channel device).
```

Comment:
#                     even though the two should be equivalent on a PXI-5142 (a 1 channel device).
 This is incorrect.  The PXI-5142 has 2 channels.

Reply:
Fixed in #1248